prog/dftb+/lib_dftbplus/initprogram.F90

```diff
@@ -2100,6 +2100,13 @@ subroutine initProgramVariables(this, input, env)
           & spin orbit calculations")
     end if
 
+    if (allocated(this%solvation)) then
+      if (this%tExtChrg .or. this%tEField) then
+        write(*,*)'External fields are not currently compatible with implicit solvents.'
```

Comment:
There is no action associated with this write statement

Reply:
Yes, that was a stupid typo on my part...